yb-voyager/cmd/importData.go

```diff
@@ -223,6 +220,10 @@ func importData(importFileTasks []*ImportFileTask) {
 	} else {
 		utils.PrintAndLog("Tables to import: %v", importFileTasksToTableNames(pendingTasks))
 		prepareTableToColumns(pendingTasks) //prepare the tableToColumns map in case of debezium
+		err = tdb.InitLiveMigrationState(migrationUUID, NUM_EVENT_CHANNELS, startClean, lo.Keys(TableToColumnNames))
```

Comment:
Should this be moved to somewhere near the beginning of `streamChanges()` function?